ruby/computer_science/common_data_structures_algorithms.md

```diff
@@ -32,11 +32,12 @@ This section contains a general overview of topics that you will learn in this l
 
 <div class="lesson-content__panel" markdown="1">
   1. Glance over the [Wikipedia entry on Data Structures](http://en.wikipedia.org/wiki/Data_structure) for a high level overview of things.
-  2. Learn about basic algorithms from Coursera's Algorithms course in [this video](http://www.youtube.com/watch?v=u2TwK3fED8A).  The first 10 minutes are really the meat of the introduction to algorithms, the rest gets more mathematical (if you're so inclined).
+  2. Learn about basic algorithms from [Coursera’s Algorithms course](http://www.youtube.com/watch?v=u2TwK3fED8A).
+  The first 10 minutes are really the meat of the introduction to algorithms, the rest gets more mathematical (if you're so inclined).
```

Comment:
No need for the line break here
```suggestion
  2. Learn about basic algorithms from [Coursera’s Algorithms course](http://www.youtube.com/watch?v=u2TwK3fED8A). The first 10 minutes are really the meat of the introduction to algorithms, the rest gets more mathematical (if you're so inclined).
```

Reply:
Hey @MaoShizhong thanks for taking the time to review my changes! The code should be clean now after I used the linter, Check it out!

Reply:
Thanks - the line break described above is still present and should be resolved, then we should be good to merge after.

Reply:
Should be good now Mao, let me know if there's anything else. Thanks for being patient!